nixos/tests/mobilizon.nix

```diff
@@ -0,0 +1,24 @@
+import ./make-test-python.nix ({ lib, ...}:
```

Comment:
I'm definitely not saying you should do it here, but maybe add a comment on running the tests from the repo at some point?
I know it would require to repackage the app, so no need to do it in this PR.

Reply:
Could you expand on what you mean by "from the repo"? Do you mean not using Elixir releases, or not using the NixOS module?

Reply:
Sorry for being vague here.

When I meant tests from the repo, I was thinking about the test suite, that is run in elixir with `mix test`. I realize though that I don't think you can do it with a release. You can probably abuse the mixRelease, and override to not output a release but just run mixTest.

Let me know if anything doesn't make sense.

Reply:
Oh, I see! This seems like something we definitely want to do. I think it would be best to do it directly in the package derivation, or even better, to do it in `mixRelease`. I don't know yet if Mobilizon wants a particular test environment

Reply:
The only problem I see with doing it in the mixRelease, is that you need to compile your stuff twice, once for the tests and once for the final release.